File: BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationException.java
package com.braintreepayments.api;

/**
 * Error class thrown when a configuration value is invalid
 */
public class ConfigurationException extends BraintreeException {
    
    ConfigurationException(String message, Throwable cause) {
        super(message, cause);
    }
    
    ConfigurationException(String message) {
        super(message);
    }
}

File: BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationException.kt
package com.braintreepayments.api

import androidx.annotation.RestrictTo

/**
 * Error class thrown when a configuration value is invalid
 */
open class ConfigurationException @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) @JvmOverloads constructor(
    message: String?,
    cause: Throwable? = null
) : BraintreeException(message, cause)
